### PR TITLE
Breaking: Bump postgres chart version to 10.3.13

### PR DESCRIPTION
**Pull request by taylorsilva**
# Why do we need this PR?
Breaking changes: https://github.com/bitnami/charts/tree/master/bitnami/postgresql#to-1000

tldr: uses helm api v2, same as our chart. Some name changes like
master -> primary, slave -> readReplicas

We should make this a major bump of the chart and link to the bitnami readme above

# Contributor Checklist

- [ ] Variables are documented in the `README.md`
- [x] Which branch are you merging into? **master**
    - `master` is for changes related to the current release of the `concourse/concourse:latest` image and should be good to publish immediately
    - `dev` is for changes related to the next release of Concourse (aka unpublished code on `master` in [concourse/concourse](https://github.com/concourse/concourse))

# Reviewer Checklist
> This section is intended for the core maintainers only, to track review progress. Please do not
> fill out this section.
- [x] Code reviewed
- [ ] Topgun tests run
- [ ] Back-port if needed
- [x] Is the correct branch targeted? (`master` or `dev`)

### CODE REVIEW

**Review by xtremerui — APPROVED** [review]

**Comment by xtremerui** [comment]
LGTM. I have added it to milestone `16.0.0` for next major release.
